app/Http/Controllers/SubscriberController.php

```diff
@@ -104,6 +104,15 @@ public function store(Request $request)
         $selectedLists[] = $allCategory->first()->id;
         $subscriber->lists()->attach($selectedLists);
 
+        if (isApi()) {
+            $data = [
+                'data' => [
```

Comment:
I don't think we need this key at all. We can just return the subscriber information
```php
return response()->json($subscriber);
```